localcache.go

```diff
@@ -249,6 +249,21 @@ func (c *Cache) ReadFile(key string) ([]byte, error) {
 	return ioutil.ReadFile(path)
 }
 
+// Purge entry for given key if older than given age.
+func (c *Cache) PurgeKey(key string, older time.Duration) error {
+	key = hash(key, false)
+	path := filepath.Join(c.root, key[:2], key)
+	_, err := os.Stat(path)
```

Comment:
The Stat() is usually unnecessary, just do the Readlink (or whatever) and check for `os.IsNotExist()`.

Reply:
yes, sorry. done.